packages/@vue/cli-plugin-typescript/generator/index.js

```diff
@@ -24,42 +23,6 @@ module.exports = (api, {
     })
   }
 
-  if (tsLint) {
-    api.extendPackage({
-      scripts: {
-        lint: 'vue-cli-service lint'
-      }
-    })
-
-    if (!lintOn.includes('save')) {
-      api.extendPackage({
-        vue: {
-          lintOnSave: false
```

Comment:
I think [one of the tests](https://circleci.com/gh/vuejs/vue-cli/18330?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) fails because this position no longer exists 🤔